composer.json

```diff
@@ -29,6 +29,8 @@
         "symfony/http-kernel": "^4.4",
         "symfony/intl": "^4.4",
         "symfony/templating": "^4.4",
+        "twig/extra-bundle": "^3.0",
```

Comment:
Do we need the extra-bundle?

Reply:
The only value this package is adding is to provide the service "[twig.extension.intl](https://github.com/sonata-project/SonataIntlBundle/pull/302/files#diff-601e179b161e2c38580772846d307576R29)".
So, if we won't to register the service by ourselves, we need it.

Reply:
To me it is a bit strange to rely on a new dependency that has to be activate on the projects of our users, same as the pr for admin bundle. I have several projects where I started with the initial versions of sonata 3.0 and sf 2.8 and they are currently on 4.4 but not using symfony flex. Having to manually register this bundle after upgrade means a Bc break to me.

Just a comment, not blocking the PR tho..

Reply:
I agree about your concern. What do you propose? Should we register a service with the same name or should we use other name?
I haven't strong reasons to fight against creating a new service instance, since we have already no choice when some options are in use:
https://github.com/sonata-project/SonataIntlBundle/blob/e004e33d3f29306da06125c78cb2234985656e07/src/Templating/Helper/NumberHelper.php#L477

Reply:
Maybe it could be nice to check if TwigExtraBundle is enabled. If not, register extension manually. wdyt?

Reply:
In order to avoid edge cases, I took a slightly different approach in order to avoid dealing with the original service:
https://github.com/sonata-project/SonataIntlBundle/blob/7a062b36ebf361ce861cd28d6f7d847c97cc75e0/src/DependencyInjection/SonataIntlExtension.php#L118-L138
Please, tell me if it covers your expectations.